x/epoching/types/msg.go

```diff
@@ -8,6 +8,7 @@ import (
 // ensure that these message types implement the sdk.Msg interface
 var (
 	_ sdk.Msg = &MsgWrappedDelegate{}
+	_ sdk.Msg = &MsgWrappedEditValidator{}
```

Comment:
how abt wrapped staking param update?

Reply:
good catch, add in https://github.com/babylonlabs-io/babylon/pull/436/commits/3d78a36883d46477335c6a9785d2ab7241dea3c0